### DISCUSSION TITLE
Remove unwanted charchers from the start of tags/fields

### DISCUSSION THREAD

**Discussion by frankwoconnor** [beetbox/beets#4918] [General]
Hope this is OK, not a question, but just sharing something helpful that I had been trying to do for a while.
I was looking to tidy up some fields by removing spaces or dashed from the start of a field.

Two things I didn't know.
1. It's possible to use modifiers for fields as well as paths, e.g. %lowercase, or %left and % right, etc.
2. There is no way to get the length of the value in a field, however %right accepts negative numbers.

So to remove all leading spaces in the field albumartist for all tracks use the command below:

`beet modify  albumartist='%right{$albumartist,-1}' albumartist::'^\s ' 
`

So to remove all leading '-' in the field albumartist for all tracks use the command below:

`beet modify  albumartist='%right{$albumartist,-1}' albumartist::'^- ' 
`

Hope this is useful to someone help, helped me out a lot.

Ps - running:
beets version 1.6.1
Python version 3.9.9

**Comment by sampsyo** [comment]
Super helpful; thanks!!
